{
  "url": "https://api.github.com/repos/facebook/react/issues/2102",
  "repository_url": "https://api.github.com/repos/facebook/react",
  "labels_url": "https://api.github.com/repos/facebook/react/issues/2102/labels{/name}",
  "comments_url": "https://api.github.com/repos/facebook/react/issues/2102/comments",
  "events_url": "https://api.github.com/repos/facebook/react/issues/2102/events",
  "html_url": "https://github.com/facebook/react/issues/2102",
  "id": 41355555,
  "node_id": "MDU6SXNzdWU0MTM1NTU1NQ==",
  "number": 2102,
  "title": "Rendering the same instance of a component doesn't throw anymore in 0.11.x",
  "user": {
    "login": "ryanseddon",
    "id": 143402,
    "node_id": "MDQ6VXNlcjE0MzQwMg==",
    "avatar_url": "https://avatars1.githubusercontent.com/u/143402?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/ryanseddon",
    "html_url": "https://github.com/ryanseddon",
    "followers_url": "https://api.github.com/users/ryanseddon/followers",
    "following_url": "https://api.github.com/users/ryanseddon/following{/other_user}",
    "gists_url": "https://api.github.com/users/ryanseddon/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/ryanseddon/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/ryanseddon/subscriptions",
    "organizations_url": "https://api.github.com/users/ryanseddon/orgs",
    "repos_url": "https://api.github.com/users/ryanseddon/repos",
    "events_url": "https://api.github.com/users/ryanseddon/events{/privacy}",
    "received_events_url": "https://api.github.com/users/ryanseddon/received_events",
    "type": "User",
    "site_admin": false
  },
  "labels": [

  ],
  "state": "closed",
  "locked": false,
  "assignee": null,
  "assignees": [

  ],
  "milestone": null,
  "comments": 1,
  "created_at": "2014-08-28T04:58:42Z",
  "updated_at": "2014-08-28T05:00:57Z",
  "closed_at": "2014-08-28T05:00:57Z",
  "author_association": "CONTRIBUTOR",
  "body": "So in our app we had some unit tests assuming that if we try and render the same component instance twice it'll throw. Upgrading to 0.11.1 doesn't seem to be the case is this intentional or a regression?\n\nSee reduced test cases below:\n\nhttp://jsfiddle.net/cy4o87yk/1/ - 0.11.0 works fine no errors\nhttp://jsfiddle.net/swoxjf13/1/ - 0.10.0 throws error\n",
  "closed_by": {
    "login": "ryanseddon",
    "id": 143402,
    "node_id": "MDQ6VXNlcjE0MzQwMg==",
    "avatar_url": "https://avatars1.githubusercontent.com/u/143402?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/ryanseddon",
    "html_url": "https://github.com/ryanseddon",
    "followers_url": "https://api.github.com/users/ryanseddon/followers",
    "following_url": "https://api.github.com/users/ryanseddon/following{/other_user}",
    "gists_url": "https://api.github.com/users/ryanseddon/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/ryanseddon/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/ryanseddon/subscriptions",
    "organizations_url": "https://api.github.com/users/ryanseddon/orgs",
    "repos_url": "https://api.github.com/users/ryanseddon/repos",
    "events_url": "https://api.github.com/users/ryanseddon/events{/privacy}",
    "received_events_url": "https://api.github.com/users/ryanseddon/received_events",
    "type": "User",
    "site_admin": false
  }
}
